app-laptop/laptop-mode-tools/laptop-mode-tools-1.74.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit multilib systemd udev
+
+MY_P="${PN}_${PV}"
+
+DESCRIPTION="Linux kernel laptop_mode user-space utilities"
+HOMEPAGE="https://github.com/rickysarraf/laptop-mode-tools/wiki"
+SRC_URI="https://github.com/rickysarraf/${PN}/releases/download/${PV}/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 ppc x86"
+IUSE="+acpi apm bluetooth"
+
+RDEPEND="sys-apps/iproute2
+	sys-apps/ethtool
+	sys-apps/which
+	|| (
+		sys-apps/sdparm
+		sys-apps/hdparm
+	)
+	acpi? ( sys-power/acpid )
+	apm? ( sys-apps/apmd )
+	bluetooth? ( net-wireless/bluez:= )"
```

Comment:
Use `optfeature` from the eutils eclass in pkg_postinst. The bluetooth USE doesn't make any difference to the ebuild itself.